graphql/graphql.go

```diff
@@ -1130,7 +1130,7 @@ func (p *Pending) Call(ctx context.Context, args struct {
 func (p *Pending) EstimateGas(ctx context.Context, args struct {
 	Data ethapi.TransactionArgs
 }) (Long, error) {
-	pendingBlockNr := rpc.BlockNumberOrHashWithNumber(rpc.PendingBlockNumber)
+	pendingBlockNr := rpc.BlockNumberOrHashWithNumber(rpc.LatestBlockNumber)
```

Comment:
In this case(`p Pending` object) , it seems to be right to get a pending block.

Reply:
totally agree. thank you :)